specs/commandline/verify.md

```diff
@@ -16,6 +16,15 @@ Warning: The resolved digest may not point to the same signed artifact, since ta
 Successfully verified signature for <registry>/<repository>@<digest>
 ```
 
+The signed descriptor may have user defined metadata attached. If the signature for the OCI artifact contains any metadata, the output message is as follows:
+
+```text
+Successfully verified signature for <registry>/<repository>@<digest>
+
+User Metadata:
+- <key> : <value>
```

Comment:
nit: Should it be
```suggestion
User Metadata:
- <key>: <value>
```

Reply:
or will it be better if it is a table view like
```
The artifact is signed with the following user metadata.

KEY    VALUE
<key>  <value>
```

Reply:
i'm down to change it, this would make it more consistent with other commands like plugin / key list